packages/wrangler/src/pages.tsx

```diff
@@ -1059,6 +1067,33 @@ export const pages: BuilderCallback<unknown, unknown> = (yargs) => {
           });
         }
       )
+    )
+    .command("project", false, (yargs) =>
+      yargs.command(
+        "list",
+        "List your Cloudflare Pages projects",
+        (yargs) => {
+          // TODO: Does this need any options?
+          return yargs;
+        },
+        async (args) => {
+          const config = readConfig(args.config as ConfigPath);
+          const accountId = await requireAuth(config);
+          let projects = await fetchResult<any>(
+            `/accounts/${accountId}/pages/projects`
+          );
+
+          projects = projects.map((project: any) => {
+            return {
+              "Project Name": project.name,
+              "Project Domains": `${project.domains.join(", ")}`,
+              Git: !!project.source,
+              "Last Modified": project.latest_deployment.modified_on,
```

Comment:
Maybe want to use the same timeago util as you have in the deployment list command?

Reply:
Oh yeah, forgot about that here for some reason 😅 Good catch!